src/Resources/contao/dca/tl_page.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+
+/**
+ * Palettes
+ */
+$GLOBALS['TL_DCA']['tl_page']['palettes']['root'] = str_replace(
+    '{layout_legend}',
+    '{registration_legend:hide},auto_activate_registration,auto_activate_where,auto_login_registration,auto_login_activation;{layout_legend:hide}',
```

Comment:
Should the fields `auto_activate_where` and `auto_login_registration` be in a subpalette? Cause they don't make sense if `auto_activate_registration` is not checked?

Reply:
As stated above, I guess we can abandon `auto_activate_where`. What do you think? Otherwise we should make it compatible with the doctrine query builder.

Reply:
fine with me, we can always add it later again 😊